lib/index.js

```diff
@@ -1,18 +1,7 @@
 /*!
- * Copyright (c) 2020-2022 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2020-2024 Digital Bazaar, Inc. All rights reserved.
  */
-import credentialsContext from 'credentials-context';
+import {contexts} from '@digitalbazaar/credentials-context';
 import {jsonLdDocumentLoader} from '@bedrock/jsonld-document-loader';
 
-const contextModules = [
-  credentialsContext
-];
-
-for(const {contexts} of contextModules) {
-  for(const [contextUrl, context] of contexts) {
-    jsonLdDocumentLoader.addStatic(contextUrl, context);
-  }
-}
-
-const {constants} = credentialsContext;
-export {constants};
```

Comment:
Hmm, I think we need these exported.

Reply:
I missed where these were used.  Changing the style a bit to export a different API to access similar info in a hopefully more flexible way.